src/index.scss

```diff
@@ -9,6 +9,7 @@ body {
 }
 
 .hero {
+  position: relative;
```

Comment:
Why? 🤔 
Modal should be relative to document body (which is default behaviour)

Reply:
Changed